SashaZaitsev/HW3/03.built-in data types.py

```diff
@@ -0,0 +1,38 @@
+# built-in data types
+better = 0
+never = 0
+iss = 0
+
+line_zen_python = input('Введіть рядок з "Zen of Python":').lower()
+for i in line_zen_python.split():
+    if i == 'better':
+        better += 1
+    elif i == 'never':
+        never += 1
+    elif i == 'is':
+        iss += 1     
+    else:
+        pass
+
+print(f'Всього "better" в рядку: {better}\nВсього "never" в рядку: {never}\nВсього "is" в рядку: {iss}')
+
+line_zen_upper = line_zen_python.upper()
+print(f'Рядок в Uppercase: {line_zen_upper}')
+replase = line_zen_python.replace('i', '&')
+print(f'Рядок із заміною літери "i": {replase}')
+
+
+
+num = int(input("Введіть чотрицифрове натуральне число: "))
+
```

Comment:
look at this version of code 
value_int = int(input('Enter 4-digit number:'))

v_int = value_int

multi = 1

while True:
    multi *= v_int % 10
    v_int = v_int // 10
    if v_int // 10 == 0:
        multi *= v_int
        break
print(f"Multiplication of digits = {multi}")
print(f"Revers of number = {str(value_int)[::-1]}")
print(f"Sorted of number = {''.join(sorted(str(value_int)))}")